src/main/java/com/minecolonies/util/StructureWrapper.java

```diff
@@ -564,19 +444,19 @@ public boolean findNextBlockToClear()
                 return false;
             }
         }
-        //Check for air blocks and if blocks below the hut are different from the schematicWorld
-        while ((worldBlockAir() || doesSchematicBlockEqualWorldBlock()) && count < Configurations.maxBlocksCheckedByBuilder);
+        //Check for air blocks and if blocks below the hut are different from the structure
+        while ((worldBlockAir() || doesStructureBlockEqualWorldBlock()) && count < Configurations.maxBlocksCheckedByBuilder);
 
         return true;
     }
 
     private boolean isAirBlock()
     {
-        return getBlock() == Blocks.air;
+        return getBlock() == Blocks.AIR;
     }
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)